### ISSUE TITLE
[bug] Throughput with place-and-adjust assumes a stack size of 1

### ISSUE THREAD

**Issue by chunkybanana** [JanSharp/quick-adjustable-inserters#1]
When holding an inserter and using place-and-adjust, the initial throughput estimation will assume a stack size of 1. Expected (modifying existing inserter):
<img width="342" alt="Screenshot 2024-02-16 at 5 59 23 PM" src="https://github.com/JanSharp/quick-adjustable-inserters/assets/62921243/53993b29-f64c-4ccb-9a93-7eb42f182bc2">

Actual (place-and-adjust):
<img width="325" alt="Screenshot 2024-02-16 at 6 00 45 PM" src="https://github.com/JanSharp/quick-adjustable-inserters/assets/62921243/7d9f8965-d710-410d-b91d-c0627eb81b4b">

(note: this is an inserter crane with stack size 250)

**Comment by JanSharp** [comment]
Damn, I thought I was handling this well enough for it to not be noticed. Basically it is using [LuaEntity::inserter_target_pickup_count](https://lua-api.factorio.com/latest/classes/LuaEntity.html#inserter_target_pickup_count), which seems like a really smart idea, except that this variable only updates when the inserter starts a swing, or something along those lines. I've not figured out the exact timing as to when it updates, but it does tend to be outdated, especially when placing new inserters or researching a new stack size tech level.

I've already implemented manually calculating the real stack size, but I've not used it everywhere since it is more expensive than using the built in variable. I'll likely go ahead and use that calculation basically everywhere unless I can find a better pattern where I can reliably use `inserter_target_pickup_count`.

Long story short, thank you for the report, I'll look into it more and fix it :+1:

**Comment by JanSharp** [comment]
I decided to do some evil cherry picking to get this out sooner. Should be fixed with version 1.1.4, thanks again for the report :)
